docs/src/routes/docs/components/tab.svx

```diff
@@ -0,0 +1,98 @@
+---
+name: Tab
+relatedComponents:
+  - name: Tabs
+    link: ./docs/components/tabs
+---
+
+<script>
+  import { Tab } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+</script>
+
+# Tab
+
+Allows for finer control over individual tabs when the default [==`Tabs`==](./docs/components/tabs) is not good enough.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Tab value="page1" name="nav1">
+      Components
+    </Tab>
+    <Tab value="page2" name="nav1" disabled>
+      Installation
+    </Tab>
+    <Tab value="Showcase" name="nav1" />
+  </div>
+  <div slot="source">
+
+```svelte
+<Tab value="page1" name="nav1">
+  Components
+</Tab>
+<Tab value="page2" name="nav1" disabled>
+  Installation
+</Tab>
+<Tab value="Showcase" name="nav1" />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`ripple`** | `true` | `boolean` | Adds a ripple to the tab when clicking on it. |
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| ==**`value`**==&nbsp;<span class="required">*</span> |  | `string` | The value assigned to the `<input type="radio">`. Check [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#Value) for more information. |
+| **`group`** | `null` | `string` | The name used to group several `<Tab>`s together. |
```

Comment:
Wait, no, that's not what `group` is, that's `name`. `group` is something you can bind to to get the value of the currently selected radio button in the name group

Reply:
Which makes me wonder why are we not explicit about having the `name` prop with the tabs. I think we should be

Reply:
We are explicit about having the `name` prop with the tabs. Check https://github.com/illright/attractions/blob/docs/tabs/docs/src/routes/docs/components/tabs.svx#L51

Reply:
Yeah, but not the `Tab`, which is strange. Best to be explicit everywhere